src/app/GitUI/UserControls/FilterToolBar.cs

```diff
@@ -24,6 +25,9 @@ public FilterToolBar()
         {
             InitializeComponent();
 
+            tscboBranchFilter.Size = new Size(DpiUtil.Scale(tscboBranchFilter.Size.Width), tscboBranchFilter.Size.Height);
+            tstxtRevisionFilter.Size = new Size(DpiUtil.Scale(tstxtRevisionFilter.Size.Width), tstxtRevisionFilter.Size.Height);
```

Comment:
Any reason we can't do the following?
```suggestion
            tscboBranchFilter.Size = DpiUtil.Scale(tscboBranchFilter.Size);
            tstxtRevisionFilter.Size = DpiUtil.Scale(tstxtRevisionFilter.Size);
```

Reply:
I had it the way you showed at first, but when comparing with unscaled version I noticed that the toolbar was getting higher for no good reason and thus switched to only scaling the width.

When it gets higher it's with `DpiUtil.Scale(tscboBranchFilter.Size)` type of scaling

https://github.com/user-attachments/assets/b3f2c48e-da09-4614-a510-0a5b082d9134

Reply:
Please add a comment explaining this, otherwise it's a hidden knowledge. Thanks